.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Package
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Build and publish Python package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install pypa/build
+        run: |
+          python -m pip install --upgrade setuptools pip wheel
+          python -m pip install build --user
+      - name: Build a binary wheel and a source tarball
+        run: python -m build --sdist --wheel --outdir dist/ .
+      - name: Publish package to test PyPI
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+      - name: Publish package to PyPI
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
should this also validate that the branch is master?
i.e. is it possible to tag something on a branch, and that would trigger this action?

same q. applies to the action we created on luma.core?

Reply:
@rm-hull no idea. You can limit who can push tags, so technically it's only you that could make this mistake ;)

Reply:
I just added 0d82366 so that this workflow only ever runs on master .. I think this is what i was getting at in my original comment